tests/script/test_realm.py

```diff
@@ -82,6 +82,39 @@ async def test_realm_realmCreated_sandbox(websocket, context_id):
     } == response
 
 
+@pytest.mark.asyncio
+async def test_realm_realmCreated_worker(websocket, context_id, html):
+    worker_url = 'data:application/javascript,while(true){}'
+    url = html(f"<script>new Worker('{worker_url}');</script>")
+
+    await subscribe(websocket, ["script.realmCreated"])
+
+    await send_JSON_command(
+        websocket, {
+            "method": "browsingContext.navigate",
+            "params": {
+                "context": context_id,
+                "url": url,
+                "wait": "complete",
+            }
+        })
+
+    # Wait for worker to be created
+    while True:
```

Comment:
You can call twice the `wait_for_event("script.realmCreated")` and verify the second event is from realm.